PukingPredator/Assets/Scripts/Player.cs

```diff
@@ -132,4 +140,25 @@ private IEnumerator SetGroundedFalse(float delay)
         ungroundedCoroutine = null;
     }
 
+    private void AlterMovement(Item item, int isEating)
```

Comment:
using int isEating where the name describes a truthy sort of value but the particular value is actually being used for something is confusing, swap it to a boolean or enum to accurately describe what the input represents and then convert to 1 or -1 inside the function if we want to make the change by just adding or subtracting those weights.